02.calendar/calendar.rb

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env ruby
+require 'date'
+require 'optparse'
+
+# コマンドラインオプション設定(デフォルト値は今日の年と月)
+params =ARGV.getopts("m:", "y:", "m:#{Date.today.mon}", "y:#{Date.today.year}")
+input_month = params["m"].to_i
+input_year = params["y"].to_i
```

Comment:
```suggestion
month = params["m"].to_i
year = params["y"].to_i
```

inputじゃないyearやmonthがあるわけではないので省略してもよさそうです

Reply:
こちら修正しました！
https://github.com/jun-kondo/ruby-practices/pull/2/commits/0fc4e9cc84608d8cd619506e8390f204a3055884